### ISSUE TITLE
"Importing Humble Bundle ... " message won't disappear.

### ISSUE THREAD

**Issue by MarkoH01** [UncleGoogle/galaxy-integration-humblebundle#175]
**Describe the bug**
When you start Galaxy it shows a panel when the importing of other platforms is starting. When it is ready with the import the Humbla panel with the message "Importing Humble Bundle ..." stays on the screen. You can close it and working normally but it is a bit annoying since all other panels are disappearing by themselves.

**Expected behavior**
Automatic disappearence of the panel once the import is finished.

**Plugin installed**
manually from your repository releases
![Humble](https://user-images.githubusercontent.com/22550982/165599449-1b32aaf4-e10e-4d24-81c6-cc7c4fa2f1d9.jpg)

**Comment by MarkoH01** [comment]
Problem was on my side. I thought the autoupdate would download all new files which apparently wasn't the case since the issue was resolved the moment I downloaded the files from here manually.

**Comment by UncleGoogle** [comment]
I'm still not sure autoupdate on "my channel" works as intended - maybe Galaxy has enhanced logic to prevent updates from other sources. Need to test that and if not - remove that feature.

And the snackbar itself is controlled by Galaxy. I assume there may be a bug on their side that snackbar do not disappear when error is raised from plugin for `get_subscription_games`. We could check and post issue on Gog side, but I'm afraid they won't fix it as a too low priority :shrug:
